### PR TITLE
screens

### PR DESCRIPTION
**Pull request by pankaj-prog**
explore page screen:
explore page

profile page:
porfile page

post page: 
post page

 auth page: 
auth

new post: 
new post page

### CODE REVIEW

File: src/components/Post/Post.jsx [hunk 1]
```diff
@@ -14,12 +17,17 @@ const Post = () => {
               alt="...."
             />
           </span>
-          <h5 className="user-name">Pankaj Wadhwani</h5>
+          <div>
+            <h5 className="user-name">Pankaj Wadhwani </h5>
+            <span className="text-muted text-sm">@pankajw01</span>
+          </div>
```
Comment (manojsarna) [inline src/components/Post/Post.jsx:23]: connect with APIs

File: src/styles/post.css [hunk 2]
```diff
@@ -19,11 +19,13 @@
...
 
 .post-user-details {
+  align-items: center;
   display: flex;
-  gap: 8px;
+  gap: 12px;
 }
 
 .post-img {
+  cursor: pointer;
```
Comment (manojsarna) [inline src/styles/post.css:28]: Are you using your own CL for this ?

File: src/styles/profile.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.posts-type-wrapper {
+  border-radius: 1rem;
+  display: flex;
+  justify-content: center;
+  background-color: var(--COLOR-SECONDARY);
+  gap: 1rem;
+}
+
```
Comment (manojsarna) [inline src/styles/profile.css:30]: rest of the code looks great

**Review by manojsarna — APPROVED** [review]
Overall the code looks great 

UI Review : Use title attribute for built in tooltips , check cursor pointers where ever required.

File: src/styles/post.css (hunk 2 above)
Comment (pankaj-prog) [reply src/styles/post.css:28]: yes

**Review by SinchanaMS — APPROVED** [review]
I don't see any issues with the code. The UI looks good!

File: src/pages/PostPage/PostPage.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+              />
+            </span>
+            <div>
+              <span className="h5">@pankajw01</span>
+              <span className="comment">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Quia
+                quas assumenda dolores, autem doloribus iusto!
+              </span>
+            </div>
+          </li>
```
Comment (m26unkwn) [inline src/pages/PostPage/PostPage.jsx]: You can create one component of it as it repeated.

**Review by m26unkwn — APPROVED** [review]
You can go ahead with the code.

it is clean and UI is awesome.

**Review by talrejah02 — APPROVED** [review]
Nice UI.
clean code.

**Pull request merged**